src/test-utils/dom/key-value-pairs/index.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { ComponentWrapper, ElementWrapper } from '@cloudscape-design/test-utils-core/dom';
+import styles from '../../../key-value-pairs/styles.selectors.js';
+import BoxWrapper from '../box';
+
+export class KeyValuePairsPairWrapper extends ComponentWrapper {
+  findLabel(): ElementWrapper | null {
+    return this.findByClassName(styles['key-label']);
+  }
+
+  findValue(): ElementWrapper | null {
+    return this.findByClassName(styles.description);
+  }
+
+  findInfo(): ElementWrapper | null {
+    return this.findByClassName(styles.info);
+  }
+}
+
+export class KeyValuePairsColumnWrapper extends ComponentWrapper {
+  findTitle(): BoxWrapper | null {
```

Comment:
This exposes an implementation detail that title uses box. You do not want that. `ElementWrapper` will be enough

Reply:
Technically yes, `ElementWrapper` is enough, but I exposed `BoxWrapper` to keep `findComponent` usage consistent, since every other usage exposes an appropriate component type. Do you think it is necessary to make it differently for this case?

Reply:
People may start relying on `BoxWrapper` return type and you will be in problem when there will be a requirement to change this for some reason

Reply:
updated